### ISSUE TITLE
Maintainer Needed?

### ISSUE THREAD

**Issue by xloem** [gunthercox/ChatterBot#2216]
What's the state of maintenance for Chatterbot?  It seems a number of important things are unresolved without comment, and there's been only a single doc fixing commit this year.

It can often be quite reasonable to find new maintainers for projects by reaching out to PR contributors and fork owners.  There are also some looking-for-maintainer hubs out there that try to bring people in.

@gunthercox are you still on this project or do you need help?  Is anyone available to step in?

**Comment by sukadateam** [comment]
I would love to step in. I actually just started finding and trying to resolve issues.

1. pytz is supposed to be checked and installed before running. Though that is not the case. The program will still attempt to run without it. Evidence: requirements.txt /root/
2. In compat.py, time.clock is being used even though it may not be supported. Fix: change line 263 and 265 to try: and except:

**Comment by brightening-eyes** [comment]
supporting python>=3.8 and so on are the other things that should be supported

**Comment by xloem** [comment]
Anybody with a fork, feel free to mention it here as a place we could consider continuing this project.  Anyone able to find and reach out to gunthercox, this would also be great.

**Comment by brightening-eyes** [comment]
I do have a fork, but as there are a lot of pull requests are here (34), I dont think it makes sence to have another fork beside it.

**Comment by xloem** [comment]
The pull requests could of course be reopened in a new fork by interested parties.

**Comment by brightening-eyes** [comment]
I don't think that @gunthercox wants to maintain this project. if thats the case, I would like to continue it.

**Comment by xloem** [comment]
sukadateam's fork: https://github.com/sukadateam/ChatterBot
brightening-eyes' fork: https://github.com/brightening-eyes/ChatterBot

**Comment by gunthercox** [comment]
Hey 👋🏼 

I replied to a similar question earlier: https://github.com/gunthercox/ChatterBot/issues/2311#issuecomment-2676930275

It's been a while but I'm looking to start active development on this project back up again. I've been attempting to get package dependencies updated, and triage open issues in bulk. And as the next major step forward I think some experimentation into large language models is due.

I'm still working on creating a concrete roadmap for the path forward, but if anyone here is still interested I'd be thrilled to be able to include others in the planning process.

Anyone interested is welcome to let me know here and I'll try to sent up some sort of planning doc or channel (open to suggestions on the best way to go about it)
